webapp/channels/src/components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.test.tsx

```diff
@@ -95,21 +93,4 @@ describe('components/shortcuts/KeyboardShortcutsSequence', () => {
         expect(wrapper.find('.shortcut-key--tooltip')).toHaveLength(2);
         expect(wrapper.find('.shortcut-key--shortcut-modal')).toHaveLength(0);
     });
-
-    test('should create sequence with order', () => {
```

Comment:
The Team Sidebar icon was the only one that actually used value substitution, and since it doesn't use `KeyboardShortcutSequence` any more, I removed the test for it.

I just realized I forgot to remove the corresponding code for it from `KeyboardShortcutSequence` though